packages/react-components/src/components/shared-components/Value/Value.tsx

```diff
@@ -2,28 +2,24 @@ import React from 'react';
 import { round } from '@iot-app-kit/core';
 import type { Primitive } from '@iot-app-kit/core';
 
-export const Value: React.FC<{ isEnabled?: boolean; value?: Primitive; unit?: string }> = ({
-  isEnabled = true,
-  value,
-  unit,
-}) => {
-  if (!isEnabled || value == null) {
+export const Value: React.FC<{ value?: Primitive; unit?: string }> = ({ value, unit }) => {
+  if (value == null) {
     return <span data-testid='no-value-present'>-</span>;
   }
 
   if (typeof value === 'number') {
     /** Display Number */
     return (
       <>
-        {round(value)} {unit && <span className='unit'> {unit}</span>}
+        {round(value)} {unit && <small> {unit}</small>}
       </>
     );
   }
 
   /** Display String or Booleans */
   return (
     <>
-      {String(value)} {unit && <span className='unit'> {unit}</span>}
+      {String(value)} {unit && <small> {unit}</small>}
```

Comment:
Blocker: Use [`Box`](https://cloudscape.design/components/box/?tabId=api) with small variant.

Reply:
unable to as it will make the font size always the same size regardless of `fontSize` setting passed in